component/azstorage/azauthmsi.go

```diff
@@ -122,7 +122,8 @@ func (azmsi *azAuthMSI) fetchTokenFromCLI() (*common.OAuthTokenInfo, error) {
 		return nil, fmt.Errorf(msg)
 	}
 
-	log.Info("azAuthMSI::fetchTokenFromCLI : Successfully fetched token from Azure CLI : %s", output)
+	log.Info("azAuthMSI::fetchTokenFromCLI : Successfully fetched token from Azure CLI")
```

Comment:
azauthspn also has similar logs, can you sync them as well.

Reply:
Sure! Addressed it on commit https://github.com/Azure/azure-storage-fuse/pull/1328/commits/4c87610ca928728c86aa6a8261b0d02788250fb0